tasks/20230525-l2-veboost-v2/input.ts

```diff
@@ -19,4 +19,12 @@ export default {
   gnosis: {
     VotingEscrow: '0xE241C6e48CA045C7f631600a0f1403b2bFea05ad',
   },
+  // https://snowtrace.io/address/0xe241c6e48ca045c7f631600a0f1403b2bfea05ad
+  avalanche: {
+    VotingEscrow: '0xE241C6e48CA045C7f631600a0f1403b2bFea05ad',
+  },
+  // https://zkevm.polygonscan.com/address/0xe241c6e48ca045c7f631600a0f1403b2bfea05ad
+  zkevm: {
+    VotingEscrow: '0xE241C6e48CA045C7f631600a0f1403b2bFea05ad',
```

Comment:
Assuming this (OmniVotingEscrowChild ) not being verified is the standard zkevm issue?

Reply:
yes, exactly.
I've already let LZ people know about it, so they are aware. They'll retry later; otherwise we can give it a shot too.
I left a link to their repo in our Notion doc for L2 boosts.